docs/quick-start/chat-agent-docs.md

```diff
@@ -182,7 +175,7 @@ of those URLS?
 First include the URLs in the [`DocChatAgentConfig`][langroid.agent.special.doc_chat_agent.DocChatAgentConfig] object:
 
```

Comment:
Thinking about how to shorten these long names... I think it's not a good idea to expose every "special" agent at the top level, so maybe at the top of the script/examples we locally define `DocChatAgentConfig = lr.agent.special.DocChatAgentConfig`, so that we are able to use just the short name and it would look less messy.